src/apps/colophon.py

```diff
@@ -0,0 +1,42 @@
+# TAXA March 2021
+
+import streamlit as st
+import base64
+from PIL import Image
+
+
+#st.set_page_config(layout="wide")
+
+
+#image = Image.open('image/TAXA.png')
```

Comment:
I think you should use this image object instead of opening the file in the markdown. I had to include the full path (assumes that the app is run from the root directory) of `image = Image.open('src/apps/image/TAXA.png')`